### PR TITLE
Update program

### PR DESCRIPTION
**Pull request by SergeiPetrovAkvelon**

### CODE REVIEW

File: Entities/EffortReport.cs [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+                return Payment * EffortTime;
+            }
+        }
+/// <summary>
+/// Calculate the payment rate.
+/// </summary>
+/// <returns></returns>
+        public string CalculatePaymentRate()
+        {
+            return $"{Payment} {(EmployeeType == EmployeesType.Hourly ? "$ / hour" : "$ / month")}";
```
Comment (tutukin) [inline Entities/EffortReport.cs]: What happens to these two methods if we decide to add a couple of other employment types?
We'll have to add more logic in this class (breaking Open-Closed rule).

May subtyping and polymorphism be used instead?

File: Application/Enums/EmployeesType.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+namespace Payroll.Application.Enums
+{
+    /// <summary>
+    /// Type of employees.
+    /// </summary>
+    public enum EmployeesType
```
Comment (tutukin) [inline Application/Enums/EmployeesType.cs]: Reads like a hint: "Define a type instead of an Enum". Kinda code smell :-).

Comment (SergeiPetrovAkvelon) [reply Application/Enums/EmployeesType.cs]: Removed this Enum. Change logic.

File: Entities/EffortReport.cs (hunk 1 above)
Comment (SergeiPetrovAkvelon) [reply Entities/EffortReport.cs]: Made changes to follow Open-Closed principle.

**Pull request merged**